db/migrate/20220430105409_create_likes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[6.1]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
```

Comment:
- [ ]  There is no migration column added. Make sure you have proper migration created for likes

![image](https://user-images.githubusercontent.com/58504655/166104354-8f4bc50f-59ef-4a52-89f2-3304fe1a5400.png)

Reply:
please check the schema.rb

Reply:
Please check the summary of my review.